src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "kandibober";
+    private static final String VALID_PASSWORD = "ibrahim13";
+    private static final String INVALID_LOGIN = "kandi";
+    private static final String INVALID_PASSWORD = "ibr13";
+    private static final int VALID_AGE = 25;
+    private static final int INVALID_AGE = 10;
```

Comment:
You should test your solution on edge-testcases. What happens if I change the minimum age from 18 to 19? or to 17? Will your tests show that sth wrong?